DOCS-2911: Document PSP betalingen for e-Boekhouden (#1944)
MultiSafepay/docs: content/accounting/integrations/e-boekhouden/integrations/psp-betalingen.md

---
title: "PSP betalingen"
weight: 10

> meta_title: "Integrations - Accounting and bookkeeping - PSP betalingen - MultiSafepay Docs"

Reviewer: ```suggestion
meta_title: "PSP betalingen - MultiSafepay Docs"
```

logo: '/svgs/psp-betalingen.png'
layout: 'single'
title_short: "PSP betalingen"
read_more: "."
short_description: "Automatically reconcile MultiSafepay payments with sales in e-Boekhouden"
description_short: "Information about PSP betalingen's reconiliation integration for e-Boekhouden"
---

> PSP betalingen lets you connect your MultiSafepay account to e-Boekhouden. With this connection, your MultiSafepay transactions are continually imported into e-Boekhouden, allowing you to automate your reconciliation.

Reviewer: ```suggestion
PSP betalingen has developed an integration that connects your MultiSafepay account to e-Boekhouden. Your MultiSafepay transactions are continually imported into e-Boekhouden, allowing you to automate your reconciliation.
```

> For more information, see PSP&nbsp;betalingen&nbsp;–&nbsp;[Koppeling MultiSafepay- e-Boekhouden (Dutch)](https://www.webwinkelfacturen.nl/koppelingpsp.php?shopsystem=multisafepay&invoicesystem=eboekhouden).

Reviewer: ```suggestion
For more information, see PSP&nbsp;betalingen&nbsp;–&nbsp;[Koppeling MultiSafepay – e-Boekhouden (Dutch)](https://www.webwinkelfacturen.nl/koppelingpsp.php?shopsystem=multisafepay&invoicesystem=eboekhouden).
```

## Installation

To connect your MultiSafepay account to e-Boekhouden, see PSP&nbsp;betalingen&nbsp;–&nbsp;[MultiSafepay - e-Boekhouden (Dutch)](https://handleidingen.pspbetalingen.nl/handleiding-multisafepay-eboekhouden).

> For the installation, you need your MultiSafepay [website-level API key](/account/site-id-api-key-secure-code/).

Reviewer: ```suggestion
You will need your MultiSafepay [website-level API key](/account/site-id-api-key-secure-code/).
```
